Bucketeer/Sources/Internal/Evaluation/EvaluationInteractor.swift

```diff
@@ -21,85 +21,110 @@ extension EvaluationInteractor {
 
 final class EvaluationInteractorImpl: EvaluationInteractor {
 
-    private static let userEvaluationsIdKey = "bucketeer_user_evaluations_id"
-
     let apiClient: ApiClient
-    let evaluationDao: EvaluationDao
-    let defaults: Defaults
+    let evaluationStorage: EvaluationStorage
+    let userEvaluationStorage: EvaluationUserDefaultDao
```

Comment:
We should access `EvaluationUserdefaultDao` through `EvalutionStorage`, as `EvaluationStorage` is an abstraction layer over the `EvaluationUserdefaultDao`.

Reply:
@yshrsmz please wait for my new review request. I have done an abstraction layer already.